config/emendatusenigmatica/strata/stone.json

```diff
@@ -0,0 +1,7 @@
+{
+    "id": "minecraft_stone",
```

Comment:
As discussed, just a reminder to expand Strata out to all of the various natural stone types such as those in Blue Skies, Twilight Forest, and those added by Quark and Create.

Also, for super secret Expert stuff, we'll need strata for Bumblezone.